### PR TITLE
FHIR-35342 Clarify expectations about scopes

### PR DESCRIPTION
**Pull request by Healthedata1**

### CODE REVIEW

File: input/pagecontent/access.md [hunk 1]
```diff
@@ -25,8 +25,10 @@ A client application gets access to a patient record by following this general s
 
 ### Scopes
 
-Scopes work as described in the SMART on FHIR specification, but note that many servers limit a client to the scopes approved on its registration, 
-and/or ignore the requested scopes at the initiation of the stand-alone launch.
+Scopes work as described in the SMART on FHIR specification. Servers **MAY** limit clients' scopes to those configured at registration time. Servers **SHOULD** allow users to select a subset of the requested scopes at the approval time.
+
+The app should inspect the returned scopes and accommodate the differences from the scopes it requested and registered.
```
Comment (jmandel) [inline input/pagecontent/access.md:30]: Is this "should" intentionally non normative?

Comment (Healthedata1) [reply input/pagecontent/access.md:30]: not indicated in resolution description conformance verb.  I interpreted as guidance. if you think it should be a SHOULD I can change it.

Comment (Healthedata1) [reply input/pagecontent/access.md:30]: I highlighted it in informational blue:

Comment (Healthedata1) [reply input/pagecontent/access.md:30]: After sleeping on it, I will make the should a SHOULD

**Pull request merged**